src/cockpit/responses.py

```diff
@@ -0,0 +1,75 @@
+# This file is part of Cockpit.
+#
+# Copyright (C) 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from typing import Optional
+
+TEMPLATE: str = """
```

Comment:
FTR, we touch this file so rarely, I'm totally fine with just embedding it here. For the C bridge we have ./src/common/fail.html and stringify/compile it in at build time, which is a tad nicer -- but such build hackery really isn't something that I'd like to do, and I'm 99% sure @allisonkarlitskaya agrees with me.

So this is just a temporary duplication, fail.html can go once we kill the C bridge.